e2e/cypress/integration/errorRedirect.spec.js

```diff
@@ -0,0 +1,21 @@
+const nextjsBaseUrl = Cypress.env("nextjsBaseUrl");
+
+it(`redirect to 404`, () => {
+  // this is a normal request for a non existing work - we expect nothing serious to happen (200)
+  cy.request(
+    `${nextjsBaseUrl}/materiale/deutsches-dichten-und-denken-vom-mittelalter-zur-neuzeit-1270-1700-_gunther-muller/work-of%3A800010-katalog%3A99122644637705763`
+  )
+    .its("headers")
+    .then((responseHeaders) => {
+      console.log(responseHeaders);
+      expect(responseHeaders).to.have.property("x-powered-by", "Next.js");
+    });
+  // this is a bot request (?isBot=true) - we expect a 404 redirect
+  cy.request({
+    url: `${nextjsBaseUrl}/materiale/deutsches-dichten-und-denken-vom-mittelalter-zur-neuzeit-1270-1700-_gunther-muller/work-of%3A800010-katalog%3A99122644637705763?isBot=true`,
```

Comment:
i stedet for at sætte isBot i url, så imiter hellere en bot via user-agent
headers: {
        "User-Agent":
          "Mozilla/5.0 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)",
      },